src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -1960,6 +2031,11 @@ mgp_error mgp_vertex_copy(mgp_vertex *v, mgp_memory *memory, mgp_vertex **result
 
 void mgp_vertex_destroy(mgp_vertex *v) { DeleteRawMgpObject(v); }
 
+mgp_error mgp_vertex_is_deleted(mgp_vertex *v, int *result) {
+  *result = IsDeleted(v);
```

Comment:
It can happen that we have some bug here, and we are not aware of it, and something may throw an exception, so I would use here wrap exceptions function in any case

Reply:
Thanks, added